src/post.py

```diff
@@ -7,14 +7,13 @@
 import random
 
 class Post():
-    def __init__(self, author: str = "No Author", contents: str = "CAW CAW CAW", points: int = -1):
+    def __init__(self, idnum: int = -1, author: str = "No Author", contents: str = "CAW CAW CAW", points: int = -1):
```

Comment:
```suggestion
    def __init__(self, id: int = -1, author: str = "No Author", contents: str = "CAW CAW CAW", points: int = -1):
```
I just prefer this naming

Reply:
`id` is a reserved keyword in Python, which is why I didn't use `id` to begin with. I agree that `id` is better, but would rather not appropriate a python keyword (even though it is technically legal.